packages/webapp/src/components/forms/AddClientForm/index.tsx

```diff
@@ -32,6 +32,16 @@ interface AddClientFormProps {
 	closeForm?: () => void
 }
 
+const lastPreferredLanguageOption =
+	CLIENT_DEMOGRAPHICS.preferredLanguage.options[
+		CLIENT_DEMOGRAPHICS.preferredLanguage.options.length - 1
+	]
+const lastRaceOption = CLIENT_DEMOGRAPHICS.race.options[CLIENT_DEMOGRAPHICS.race.options.length - 1]
```

Comment:
Should we consider using a util: maybe `import last from 'lodash/last` or we define a new util

Reply:
Done, this will make the code more readable.
I moved those `const` out of the component because they were an easy thing to lighten it.